diff --git a/node_modules/jquery/dist/jquery.js b/node_modules/jquery/dist/jquery.js
index 2170756..7bee3e0 100644
--- a/node_modules/jquery/dist/jquery.js
+++ b/node_modules/jquery/dist/jquery.js
@@ -205,7 +205,7 @@ jQuery.fn = jQuery.prototype = {
 
 	map: function( callback ) {
 		return this.pushStack( jQuery.map( this, function( elem, i ) {
-			return callback.call( elem, i, elem );
+			return callback.call( elem, elem, i );
 		} ) );
 	},
 
@@ -378,13 +378,13 @@ jQuery.extend( {
 		if ( isArrayLike( obj ) ) {
 			length = obj.length;
 			for ( ; i < length; i++ ) {
-				if ( callback.call( obj[ i ], i, obj[ i ] ) === false ) {
+				if ( callback.call( obj[ i ], obj[ i ], i ) === false ) {
 					break;
 				}
 			}
 		} else {
 			for ( i in obj ) {
-				if ( callback.call( obj[ i ], i, obj[ i ] ) === false ) {
+				if ( callback.call( obj[ i ], obj[ i ], i ) === false ) {
 					break;
 				}
 			}
@@ -4181,7 +4181,7 @@ var access = function( elems, fn, key, value, chainable, emptyGet, raw ) {
 				fn(
 					elems[ i ], key, raw ?
 					value :
-					value.call( elems[ i ], i, fn( elems[ i ], key ) )
+					value.call( elems[ i ], fn( elems[ i ], key ), elems[ i ], i )
 				);
 			}
 		}
